test/e2e/framework/runtimeobject/runtimeobject.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package runtimeobject
```

Comment:
Guessing we did this in the same PR because of dependency in framework/pod?

Reply:
yeah. the methods defined in that pkg are only used by pods, as far as i could tell.

You mentioned i should put these changes in a separate PR: all the changes elated to this pkg are within the second commit of this pr already, is this ok or should i make a separate PR out of that commit and just keep the 1st commit for this PR?.

Reply:
yeah if we can keep runtimeobj changes in a separate PR that would be good

Reply:
done. Removed the refactoring of the runtimeobj pkg off of this PR